packages/fxa-settings/src/components/ChangePassword/index.tsx

```diff
@@ -8,6 +8,7 @@ import FlowContainer from '../FlowContainer';
 import PasswordInput from '../PasswordInput';
 import LinkExternal from 'fxa-react/components/LinkExternal';
 
+// eslint-disable-next-line no-empty-pattern
 export const ChangePassword = ({}: RouteComponentProps) => {
```

Comment:
The object of type `RouteComponentProps` is apparently needed in order to match the function signature expected when calling this, but we don't use any of the properties from it so it violates the eslint rule 😒

Reply:
They used this in the react reach documentation, but maybe there is a better way to do this? My typescript foo still WIP :/